src/url_validation.js

```diff
@@ -0,0 +1,15 @@
+const { body, validationResult } = require("express-validator");
+
+// Define an array of middleware functions to validate the URL
+const validateURL = [
+  body("url").isURL(),
```

Comment:
You can also make sure the param 'url' is present, because it's the only thing required.